src/usePermission.ts

```diff
@@ -30,13 +30,16 @@ const usePermission = (permissionDesc: PermissionDesc): State => {
   };
 
   useEffect(() => {
-    navigator.permissions
-      .query(permissionDesc)
-      .then(status => {
-        permissionStatus = status;
-        changeState();
-      })
-      .catch(noop);
+    // Permissions API is not available in every browser
+    if (navigator.permissions && navigator.permissions.query) {
```

Comment:
Can be simplified `navigator.permissions?.query`

Reply:
Looks like this was already updated to

```
if ("permissions" in navigator) {
```